test/e2e/scenarios/terms-and-conditions-page/termAndConditionPage.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+const paths = require('paths');
+const termsAndConditionsContent = require('terms-and-conditions-page/content.en.json');
+
+Feature('Terms and Conditions Page');
+
```

Comment:
Please remove all tabs, as previously mentioned we only do spaces. Thanks.
<img width="917" alt="no-tabs-please" src="https://user-images.githubusercontent.com/964268/33432161-dc3bbf86-d5ce-11e7-9b13-8da411644a03.png">

Reply:
Tabs show up as lines in Webstorm, spaces show up as dots. This is why the indentation is out on the PR.

Reply:
*cough* IntelliJ *cough* @iguannago

Reply:
you guys finally made me move to Webstorm!

Reply:
You know it makes sense.